src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx

```diff
@@ -265,8 +265,8 @@ Here are the OpenTelemetry data types we support and their associated mappings.
       <td>❌</td>
     </tr>
     <tr>
-      <td>[Array of primitives (homogeneous)](https://github.com/open-telemetry/opentelemetry-specification/blob/e13990125809a4ecc482553c32d05e38bb414acb/specification/common/common.md#attributes)</td>
-      <td>❌</td>
+      <td>[Array of primitives (homogeneous)](https://github.com/open-telemetry/opentelemetry-specification/blob/e13990125809a4ecc482553c32d05e38bb414acb/specification/common/common.md#attributes). Must be non-nested and 64 or less elements.</td>
+      <td>✅</td>
```

Comment:
I suspect that we will also want to update the Logs and Traces sections, but other teams are working on that so I'll let them add whatever details they need.

Reply:
Logging will have one order of the same docs, please!